de/README.md

```diff
@@ -8,7 +8,7 @@ actionLink: /de/book/
 features:
   - title: Pipelines um jedes Betriebssystem zu steuern
     details: Nu läuft auf Linux, macOS und Windows. Einmal lernen - überall verwenden.
-  - title: Alles sind Daten.
+  - title: Alles ist Daten.
```

Comment:
I think a bit more comprehensible translation would be:

Alles besteht aus Daten.

Reply:
I'm a native German speaker. The originally suggested "Alles ist Daten" sounds awful and honestly doesn't make sense.

*(Alles is not singular. It's everything. So *sind* makes more sense.)*

I think the original or what @ehasnain suggested is much better. With the *details* context, what @ehasnain suggested is a good and fitting title.

*Honestly, I think the original EN text could be improved in the same way as here. The statement is too generic and text-shell-context-knowledge dependent. But we don't have to raise that here in this specific PR.*